lib/ddtrace/contrib/cucumber/patcher.rb

```diff
@@ -0,0 +1,45 @@
+require 'ddtrace/contrib/patcher'
+require 'ddtrace/contrib/cucumber/instrumentation'
+
+module Datadog
+  module Contrib
+    module Cucumber
+      # Patcher enables patching of 'cucumber' module.
+      module Patcher
+        include Contrib::Patcher
+
+        module_function
+
+        def patched?
+          done?(:cucumber)
+        end
+
+        def target_version
+          Integration.version
+        end
+
+        # patch applies our patch
+        def patch
+          do_once(:cucumber) do
```

Comment:
I believe you were referencing one of the older integrations to guide you here.

Take a look at one of the newer ones: https://github.com/DataDog/dd-trace-rb/blob/52926d7e911c83bc7fbea7ba1ee5169d5fb897c9/lib/ddtrace/contrib/sneakers/patcher.rb

Only `target_version` and `patch` are strictly required anymore.